digitalpy/core/zmanager/impl/async_action_mapper.py

```diff
@@ -141,7 +172,12 @@ def process_action(
 
         return routing_subscriber
 
-    def submit_request(self, request):
+    def submit_request(self, request: Request):
+        """send a request object to the routing publisher
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D403 First word of the first line should be properly capitalized